### ISSUE TITLE
update link is broken

### ISSUE THREAD

**Issue by 4TT1L4** [eclipse-color-theme/eclipse-color-theme#281]
The update link seems to be broken.

Due to recent changes the update link should using the new structure.

The old URLs like; https://eclipse-color-theme.github.com/update - won't be redirected since 15.04.2021.

The old style links should be replaced by the new ones.

In the case of the plugin, the following link should be used:

https://eclipse-color-theme.github.io/update/

This link should be used in several places, including the README.md file and the eclipse marketplace.

Currently both of these are broken.

The old link leads to the following error message, where the issue is described in detail:

![image](https://user-images.githubusercontent.com/2914096/115354857-5fe89880-a1ba-11eb-916a-5a8fae7786d2.png)

**Comment by davidebaldini** [comment]
Is there a workaround to easily install the plugin before a fix is committed to this repo?

**Comment by 4TT1L4** [comment]
Yes. There is a pretty straightforward workaround for this issue.

Just copy the new link to the eclipse window (under help/install new software):

![image](https://user-images.githubusercontent.com/2914096/115934437-4df44780-a491-11eb-955d-524c60d40d35.png)

Add a new  repository with the updated/new URI:

https://eclipse-color-theme.github.io/update/

![image](https://user-images.githubusercontent.com/2914096/115934371-2c935b80-a491-11eb-9452-efa4ac51bbda.png)

This way the plugin appears as expected and it can be installed:

![image](https://user-images.githubusercontent.com/2914096/115934401-3c12a480-a491-11eb-8d9c-36c01c81358c.png)

I have just double-checked it for you, and it works like a charm.

**Comment by davidebaldini** [comment]
@BujakiAttila I confirm this method works and the plugin appears to install properly, thanks for putting your time in the answer.

**Comment by neo-turak** [comment]
2022 still has not fixed this issues yet.

**Comment by JayWett** [comment]
Unfortunately, while the plugin installs correctly, you cannot preview the themes because the URLs do not use HTTPS. If you select a theme and apply, it does load it. You will just have to "preview" it live.

**Comment by moeenio** [comment]
I have created a new working listing on the Eclipse Marketplace : https://marketplace.eclipse.org/content/eclipse-color-theme-0

If a maintainer of this repo wants access to edit this listing, they should at-mention me here or contact me at locness3 at e dot email.

**Comment by AllisonXY** [comment]
I downloaded the color theme with the updated link. I can't preview the theme with the page saying "Can’t reach this page. •Make sure the web address http://www.eclipsecolorthemes.org is correct". And I can't even click "Apply" to see the theme taking effect. Does anyone know how to fix this?

**Comment by moeenio** [comment]
@AllisonXY you can apply color schemes without having that page working.

**Comment by AllisonXY** [comment]
> @AllisonXY you can apply color schemes without having that page working.

I tried again today and it works! Thanks a lot
